README.md

```diff
@@ -96,15 +102,15 @@ A note regarding "open a shell" in the following sections: this implies that the
 #### Iterative Python Development
 
 1. Open a shell in the PyPRT git root.
-1. First time only: run `pipenv install` to get all required Python packages.
-1. Run `pipenv shell` to activate the required Python packages.
+1. First time only: setup a virtual Python environment with build dependencies for PyPRT, see "Build Python Wheel" above.
```

Comment:
Maybe replace the "above" by "[above] (#build-python-wheel)" (without the white space).

Reply:
good idea, done.